src/components/ui/input.jsx

```diff
@@ -0,0 +1,21 @@
+import PropTypes from "prop-types";
+
+export default function Input({ type, placeholder }) {
+  return (
+    <input
+      type={type}
+      placeholder={placeholder}
+      className="rounded border-[1px] border-solid border-black/40 text-sm p-1 h-fit outline-none disabled:cursor-not-allowed disabled:opacity-50"
```

Comment:
If if create a component like this,
```jsx
<Input value="Joseph" />
```

I will expect to see an input with `Joseph` already prefilled into the input, but that is not what I get with your implementation.

Reply:
spread operator on the props will handle this

Reply:
@cremirdevio  This issue has been resolved with the use of spread operator on the props.

resolves #2